ban-cooking.lua

```diff
@@ -2,6 +2,33 @@
 -- based on ban-cooking.rb by Putnam: https://github.com/DFHack/scripts/pull/427/files
 -- Putnams work completed by TBSTeun
 
+local options = {}
+
+local argparse = require('argparse')
+local commands = argparse.processArgsGetopt({...}, {
+    {'h', 'help', handler=function() options.help = true end},
+    {'v', 'verbose', handler=function() options.verbose = true end},
+})
+
+if options.help == true then
+    print(dfhack.script_help())
+    return
+end
+
+if commands.contains('all') then
```

Comment:
I don't think there is a `contains`, but you probably also don't need to be that general. just testing `commands[1] == 'all'` should be sufficient. You don't need to support `all` appearing later in the argument list.